content/cnbattle/drone-upx/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2016-01-01T00:00:00+00:00
+title: Drone Upx
+author: cnbattle
+tags: [ "upx" ]
+repo: "cnbattle/drone-upx"
+logo: "drone.svg"
```

Comment:
I don't think that drone.svg would be a matching logo. If UPX doesn't provide a logo find some more or less matching SVG.